k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-platform/templates/cortx-hax-svc.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.services.create -}}
```

Comment:
`cortx-platform` is installed twice as part of the current `deploy-cortx-cloud.sh` script.  This was not an issue previously because "if namespace != default" guarded the first call to create the namespace.

Reply:
Since posting this comment, there has been a change in that cortx-platform is now only helm-installed once, but this code remains until/unless we decide to remove the "create" field throughout this helm chart.

Reply:
I think it's fine, we can fix it later.